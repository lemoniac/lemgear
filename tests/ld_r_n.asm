ld A, 1
ld B, 2
ld C, 3
ld D, 4
ld E, 5
ld H, 6
ld L, 7

halt
