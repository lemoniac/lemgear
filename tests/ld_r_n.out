AF : 0100
BC : 0203
DE : 0405
HL : 0607
